### DISCUSSION TITLE
Bug Report: Issue with SSO Login via GitHub

### DISCUSSION THREAD

**Discussion by mariasantosdev** [community/community#147389] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

I encountered a bug with SSO login using GitHub. Here's the issue:

When I attempt to log in via GitHub and mistakenly enter the wrong mobile authentication code (two-factor authentication), I am redirected back to the login screen where I am prompted to enter my "Username or email address" and password again. After completing this step, a new mobile authentication code is sent.

However, instead of logging me into the intended platform (e.g., GitLab, which uses SSO via GitHub), it logs me directly into GitHub itself.

This behavior only occurs after entering the wrong mobile authentication code for the first time.

**Comment by mecodeatlas** [comment]
Thanks for posting in the GitHub Community, @mariasantosdev!

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Discussions category is solely related to conversations around the [GitHub product Discussions](https://docs.github.com/en/discussions/collaborating-with-your-community-using-discussions/about-discussions). This question should be in the General category. I've gone ahead and moved it for you. Good luck!
